src/test/java/caffeinateme/steps/Barista.java

```diff
@@ -1,17 +1,29 @@
 package caffeinateme.steps;
 
 import caffeinateme.Order;
+import caffeinateme.OrderReceipt;
 import net.serenitybdd.core.steps.ScenarioActor;
 import net.thucydides.core.annotations.Steps;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class Barista extends ScenarioActor {
 
     @Steps(shared = true)
     CoffeeOrdersClient coffeeOrders;
 
     public List<Order> pendingOrders() {
-        return coffeeOrders.getOrders();
+        return coffeeOrders.getOrders().stream()
+                .filter(order -> order.getOrderStatus() != "Complete")
+                .collect(Collectors.toList());
+    }
+
+    public void completeOrder(OrderReceipt orderReceipt) {
+        coffeeOrders.getOrders().stream().filter(order -> order.getCustomerId()==orderReceipt.getCustomerId())
```

Comment:
This would work if customerId was a number, but if it is a string it is better to use equals()